LayoutTests/editing/selection/ios/hide-selection-in-hidden-contenteditable.html

```diff
@@ -65,19 +65,25 @@
     testRunner.dumpAsText();
     testRunner.waitUntilDone();
 
+    console.log("1. Waiting for keyboard to appear");
     await UIHelper.activateAndWaitForInputSessionAt(160, 160);
     document.querySelector("#caret-rect").textContent = rectToString(await UIHelper.getUICaretViewRect());
 
+    console.log("2. Selecting all");
```

Comment:
Maybe we can give it a slightly different leveling between two tests so that when output gets mixed up, we'd know which test's step 2-5 it was?

Reply:
Good call! I added "…in subframe" to some of the messages in `hide-selection-in-hidden-contenteditable-frame.html`, both for clarity and to help differentiate the two.